src/fitness_tracker/workout_providers/intervals_icu.py

```
@@ -1,105 +1,260 @@
+                    safe_title=safe_title,
+                    used_names=used_names,
+                )
+                stage_path = stage_dir / output_name
+                try:
+                    stage_path.write_text(
+                        json.dumps(event.model_dump(mode="json"), ensure_ascii=False),
+                        encoding="utf-8",
+                    )
+                    with stage_path.open("rb") as handle:
+                        os.fsync(handle.fileno())
+                except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)